### PR TITLE
Add method for creating comments on commits

### PR DESCRIPTION
**Pull request by dhodovsk**

### CODE REVIEW

File: ogr/services/github.py [hunk 1]
```diff
@@ -155,6 +163,14 @@ def _get_all_pr_comments(self, pr_id: int) -> List[PRComment]:
             for raw_comment in pr.get_issue_comments()
         ]
 
+    def get_sha_from_tag(self, tag_name: str) -> str:
+        # TODO: This is ugly. Can we do it better?
```
[the remark this answers is not in the document]
Comment (dhodovsk) [reply ogr/services/github.py]: Unfortunately, that can't be done.
```
>>> from ogr.services.github import GithubService
>>> from ogr.services.github import GithubProject
>>> s = GithubService(token='xxx')
>>> p = GithubProject('ogr', s, 'dhodovsk')
>>> p.github_repo.get_tag("0.0.1")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Repository' object has no attribute 'get_tag'
```

[the remark this answers is not in the document]
Comment (dhodovsk) [reply ogr/services/github.py]: There is only getting tag from sha: https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.get_git_tag

[the remark this answers is not in the document]
Comment (TomasTomecek) [reply ogr/services/github.py]: how about get_release then? https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.get_release

[the remark this answers is not in the document]
Comment (dhodovsk) [reply ogr/services/github.py]: From release also I can't get info about commit unfortunatelly. committish = branch
```
>>> r = p.github_repo.get_release("0.0.1")
>>> r.
r.CHECK_AFTER_INIT_FLAG   r.get__repr__(            r.raw_data                r.update(
r.author                  r.get_assets(             r.raw_headers             r.update_release(
r.body                    r.html_url                r.setCheckAfterInitFlag(  r.upload_asset(
r.created_at              r.id                      r.tag_name                r.upload_url
r.delete_release(         r.last_modified           r.tarball_url             r.url
r.draft                   r.prerelease              r.target_commitish        r.zipball_url
r.etag                    r.published_at            r.title      
>>> r.raw_data
{'url': 'https://api.github.com/repos/packit-service/ogr/releases/14947662', 'assets_url': 'https://api.github.com/repos/packit-service/ogr/releases/14947662/assets', 'upload_url': 'https://uploads.github.com/repos/packit-service/ogr/releases/14947662/assets{?name,label}', 'html_url': 'https://github.com/packit-service/ogr/releases/tag/0.0.1', 'id': 14947662, 'node_id': 'MDc6UmVsZWFzZTE0OTQ3NjYy', 'tag_name': '0.0.1', 'target_commitish': 'master', 'name': '0.0.1', 'draft': False, 'author': {'login': 'lachmanfrantisek', 'id': 20214043, 'node_id': 'MDQ6VXNlcjIwMjE0MDQz', 'avatar_url': 'https://avatars1.githubusercontent.com/u/20214043?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/lachmanfrantisek', 'html_url': 'https://github.com/lachmanfrantisek', 'followers_url': 'https://api.github.com/users/lachmanfrantisek/followers', 'following_url': 'https://api.github.com/users/lachmanfrantisek/following{/other_user}', 'gists_url': 'https://api.github.com/users/lachmanfrantisek/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/lachmanfrantisek/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/lachmanfrantisek/subscriptions', 'organizations_url': 'https://api.github.com/users/lachmanfrantisek/orgs', 'repos_url': 'https://api.github.com/users/lachmanfrantisek/repos', 'events_url': 'https://api.github.com/users/lachmanfrantisek/events{/privacy}', 'received_events_url': 'https://api.github.com/users/lachmanfrantisek/received_events', 'type': 'User', 'site_admin': False}, 'prerelease': False, 'created_at': '2019-01-11T14:15:26Z', 'published_at': '2019-01-13T14:04:52Z', 'assets': [], 'tarball_url': 'https://api.github.com/repos/packit-service/ogr/tarball/0.0.1', 'zipball_url': 'https://api.github.com/repos/packit-service/ogr/zipball/0.0.1', 'body': '# 0.0.1\r\n\r\n- Initial release.\r\n- First version of API.\r\n- Basic Pagure Implementation.\r\n'}
```

[the remark this answers is not in the document]
Comment (TomasTomecek) [reply ogr/services/github.py]: so... what can we do, let's leave it is it is now

**Comment by TomasTomecek** [comment]
Merging because I need this one, please address the tests in a new PR

**Pull request merged**